test/FileFormats/LP/LP.jl

```diff
@@ -513,7 +510,7 @@ function test_read_model1_tricky()
     @test occursin("CON4: 1 V5 + 1 V6 + 1 V7 <= 1", file)
     @test occursin("CON1: 1 V1 >= 0", file)
     @test occursin("CON5: [ 1 Var4 ^ 2 - 1.2 V5 * V1 ] <= 0", file)
-    @test occursin("R1: 1 V2 >= 2", file)
+    @test occursin("1 V2 >= 2", file)
```

Comment:
The decision to arbitrarily give the constraint a name was ... suspect. If the user didn't provide a name. We don't need to add one.